src/components/CreateWallet.jsx

```diff
@@ -64,47 +64,389 @@ const WalletCreationConfirmation = ({ createdWallet, walletConfirmed }) => {
+    } else {
+      setFeedback(true)
+    }
+  }
+
+  if (step === 0) {
+    return (
+      <div>
+        <div className="mb-4">
+          <div>Wallet Name</div>
+          <div className="fs-4">{walletDisplayName(createdWallet.name)}</div>
         </div>
+        <div className="mb-4">
+          <Seedphrase seedphrase={createdWallet.seedphrase} isBlurred={!revealSensitiveInfo} />
+        </div>
+        <div className="mb-4">
+          <div>Password</div>
+          <div className={`fs-4${revealSensitiveInfo ? '' : ' blurred-text'}`}>
+            {!revealSensitiveInfo ? 'randomrandom' : createdWallet.password}
+          </div>
+        </div>
+        <div className="mb-2">
+          <ToggleSwitch
+            label="Reveal sensitive information"
+            onToggle={(isToggled) => {
+              setRevealSensitiveInfo(isToggled)
+              setSensitiveInfoWasRevealed(true)
+            }}
+          />
+        </div>
+        <div className="mb-4">
+          <ToggleSwitch
+            label="I've written down the information above."
+            onToggle={(isToggled) => setUserConfirmed(isToggled)}
+          />
+        </div>
+        <rb.Button variant="dark" disabled={!sensitiveInfoWasRevealed || !userConfirmed} onClick={() => setStep(1)}>
+          Next
+        </rb.Button>
       </div>
-      <div className="mb-2">
-        <ToggleSwitch
-          label="Reveal sensitive information"
-          onToggle={(isToggled) => {
-            setRevealSensitiveInfo(isToggled)
-            setSensitiveInfoWasRevealed(true)
-          }}
-        />
-      </div>
-      <div className="mb-4">
-        <ToggleSwitch
-          label="I've written down the information above."
-          onToggle={(isToggled) => setUserConfirmed(isToggled)}
-        />
+    )
+  } else {
+    return (
+      <div>
+        <div className="fs-4">Confirm seed phrase backup</div>
+        <p className="text-secondary">Enter each word in sequential order.</p>
+
+        <rb.Form noValidate>
+          <div className="container">
+            <div className="row mb-4">
+              <div className="col">
+                <rb.InputGroup />
+                <rb.FormControl
+                  type="text"
+                  placeholder="One"
+                  value={wordOne}
+                  onChange={(e) => {
+                    setFeedback(false)
+                    setWordOne(e.target.value)
+                  }}
+                  disabled={seedBackup}
+                  isInvalid={!wordOne === seedphrase[0]}
+                  isValid={wordOne === seedphrase[0]}
+                  required
+                />
+                {wordOne === seedphrase[0] && !seedBackup ? (
+                  <rb.Form.Control.Feedback>Word matches!</rb.Form.Control.Feedback>
+                ) : (
+                  <rb.Form.Control.Feedback type="invalid">Word does not match.</rb.Form.Control.Feedback>
+                )}
+              </div>
```

Comment:
Is this possible in this situation because every 2 input boxes go inside a container row `div`? So not every input gets the same code if that makes sense.